crate/operator/operations.py

```diff
@@ -527,12 +527,12 @@ async def restart_cluster(
     elif next_pod_name in all_pod_names:
         total_nodes = get_total_nodes_count(old["spec"]["nodes"], "all")
         # The new pod has been spawned. Only a matter of time until it's ready.
+        node_index = int(next_pod_name[next_pod_name.rindex("-") + 1 :])
+        node_progress = f"{node_index + 1}/{len(all_pod_uids)}"
         await send_operation_progress_notification(
             namespace=namespace,
             name=name,
-            message="Waiting for node "
-            f"{int(next_pod_name[next_pod_name.rindex('-')+1:])+1}/{len(all_pod_uids)}"
-            " to be restarted...",
+            message=f"Waiting for node {node_progress} to be restarted.",
```

Comment:
That's just because of line length limitations, now raised by the linter.
Cutting the statements in the middle of the string interpolation feels nasty, so we did it this way.